lib/country_finder.ex

```diff
@@ -31,15 +31,26 @@ defmodule CountryFinder do
       flag_png = get_in(country_map, ["flags", "png"])
       region = country_map["region"]
       capital = country_map["capital"]
-      population = country_map["population"]
+      population = add_comma(country_map["population"])
+      country_code = get_in(country_map, ["cca3"])
```

Comment:
```suggestion
      country_code = country_map["cca3"]
```
Don't use `get_in` function when it is not listed

Reply:
Done